apps/remote_control/test/lexical/remote_control/build_test.exs

```diff
@@ -205,7 +205,12 @@ defmodule Lexical.BuildTest do
       assert diagnostic.uri
       assert diagnostic.severity == :error
       assert diagnostic.message =~ ~S[undefined function doesnt_exist/0]
-      assert diagnostic.position == 2
+
+      if version_after_1_15?() do
```

Comment:
nit: you're not checking if the version is after 1.15, you're checking if you're using `with_diagnostics`

Reply:
So, Do you mean changing the name to `with_diagnostics?` is better?

Reply:
correct